openslides/users/models.py

```diff
@@ -1,10 +1,7 @@
 # TODO: Check every app, that they do not import Group or User from here.
 
-from django.contrib.auth.models import (PermissionsMixin, AbstractBaseUser,
-                                        BaseUserManager)
-
-# TODO: Do not import the Group in here, but in core.models (if necessary)
```

Comment:
Please do not remove this TODO or add a short instruction to a specific GitHub issue.